spec/javascripts/Users/permissions.spec.js

```diff
@@ -298,19 +301,19 @@ describe('ServiceAccess', function () {
     let state = {}
     let node = () => render(<ServiceAccess service={service}/>, state)
 
-    expect(node()).toContainElement(checked)
```

Comment:
Same as above comment, extract individual methods instead of adding comments.

Reply:
Just for referencing the comment that he is talking about and to be easier to find:  https://github.com/3scale/porta/pull/121#discussion_r238218639 😄